lib/puppet/type/cisco_evpn_multisite.rb

```diff
@@ -0,0 +1,60 @@
+#
+# December 2017
+#
+# Copyright (c) 2017-2018 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Puppet::Type.newtype(:cisco_evpn_multisite) do
+  @doc = %(Manages a Cisco Evpn Multisite.
+
+  cisco_evpn_multisite {'<multisite>':
+    ..attributes..
+  }
+
+  <multisite> is the Evpn Multisite Site-Id.
+
+  Example:
+    cisco_evpn_multisite {'100':
+      ensure              => present,
+      delay_restore       => '500',
+    }
+  )
+
```

Comment:
Is there a reason you don't use `self.title_patterns` for this type?

Reply:
This type has only one namevar. I think it's overkill to add title_patterns here.

Reply:
I've added a check to ensure that the multisite id is a integer and give a meaningful failure if not.